src/internal/storage/fileset/util.go

```diff
@@ -26,6 +28,7 @@ import (
 func NewTestStorage(ctx context.Context, t testing.TB, db *pachsql.DB, tr track.Tracker, opts ...StorageOption) *Storage {
 	_, chunks := chunk.NewTestStorage(t, db, tr)
 	store := NewTestStore(ctx, t, db)
+	require.NoError(t, dbutil.WithTx(ctx, db, CreatePinsTable))
```

Comment:
This should probably be in a migration.  How do we get the rest of the tables?

Reply:
This is just for storage layer testing. The appropriate tables are setup by each test component that is created (i.e. tracker, chunk storage, fileset store, etc.). There might be a better way to organize this, but that is the current pattern for storage layer testing.